### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by MichalFupso**
## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

**Review by mgleung — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,6 +7,19 @@ to make installation, upgrades, and ongoing lifecycle management of Calico and C
 
 This operator is built using the [operator-sdk](https://github.com/operator-framework/operator-sdk), so you should be familiar with how that works before getting started.
 
+## Getting Started Running Calico
+
+There are many avenues to get started running Caico depending on your situation.
```
Comment (rene-dekker) [inline README.md:12]: Caico -> Calico

File: README.md [hunk 2]
```diff
@@ -7,6 +7,19 @@ to make installation, upgrades, and ongoing lifecycle management of Calico and C
...
+- Trying out Kubernetes on a single host or on your own hardware? The [quick start guide](https://projectcalico.docs.tigera.io/getting-started/kubernetes/quickstart) will have you up and running in about fifteen minutes.
+- Running a managed public cloud? Use our guides for enabling Calico network policies.
+  - [Amazon Elastic Kubernetes Service (EKS)](https://docs.tigera.io/calico/3.25/getting-started/kubernetes/managed-public-cloud/eks)
+  - [Google Kubernetes Engine (GKE)](https://docs.tigera.io/calico/3.25/getting-started/kubernetes/managed-public-cloud/gke)
+  - [IBM Cloud Kubernetes Service (IKS)](https://docs.tigera.io/calico/3.25/getting-started/kubernetes/managed-public-cloud/iks)
+  - [Microsoft Azure Kubernetes Service (AKS)](https://docs.tigera.io/calico/3.25/getting-started/kubernetes/managed-public-cloud/aks)
+- Want to go deeper? Visit [https://docs.tigera.io/](https://docs.tigera.io/) for full documentation.
+
```
Comment (rene-dekker) [inline README.md:20]: nit: Excess empty lines

**Comment by MichalFupso** [comment]
@rene-dekker Addressed your comments

**Comment by tmjd** [comment]
These links should go to the latest docs, so they will stay up-to-date, instead of linking to a specific version of the docs.

**Review by tmjd — APPROVED** [review]
LGTM

**Pull request merged**